recipes/nodebook/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nodebook" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "b0cbb9cee2deea33b3dc3d1c195b3b96a636dcdec18201cb99eb59f9c0d93a63" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/stitchfix/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Could this be `noarch: python`? 

https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Yes, thanks, will update